ci-operator/step-registry/openshift-qe/ocp-qe-perfscale-ci-tests/openshift-qe-ocp-qe-perfscale-ci-tests-chain.yaml

```diff
@@ -2,7 +2,7 @@
 chain:
   as: openshift-qe-ocp-qe-perfscale-ci-tests
```

Comment:
Should we rename this to include `control-plane` in this PR?

Reply:
Updated